yarn.lock

```diff
@@ -1032,69 +1032,6 @@
     react-loader-spinner "^5.3.4"
     storybook-addon-next-router "^4.0.1"
 
-"@sentry/browser@7.41.0":
```

Comment:
why is all of this sentry code removed?

Reply:
honestly i was thinking the same thing! i was looking at it as well- all i did was add the cypress-dotenv-flow plugin, and this is what happened- i figured maybe it was a dependency with that plugin but that would be super weird. 
gonna try running yarn again

Reply:
yeah, it may have been some errant stuff. I'm working on some other sentry stuff now that will fix/override it anyway, so let's ship it.